src/mstop.sh

```diff
@@ -46,7 +46,7 @@ for pidfile in $pfiles; do
   signal=""
   while ps -p $pid -o args= | grep -q -e $entity $extracheck ; do
     if [[ "$counter" -gt MAX_RETRIES ]]; then
-        signal="-9"
+        signal="-3"
```

Comment:
i think this `kill -9` is the right thing to do here. this loops several times trying `kill` first, but after reaching MAX_RETRIES it gives up and kills with -9

Reply:
in my experience, the issue is that we're trying to stop everything at once. but radosgw may send osd ops during shutdown, so it'll hang if osds go down first

the correct order should probably be clients (rgw, mds, mgr) then osds then mons